.github/dependabot.yml

```
@@ -8,16 +8,23 @@ updates:
     ignore:
       - dependency-name: "avocado-framework*"
     schedule:
-      interval: daily
+      interval: "cron"
+      cronjob: "0 0 1-7,15-21 * 0"  # first and third Sunday
```

It seems this will run on more days than intended based upon https://crontab.guru/#0_0_1-7,15-21_*_0 - a link provided at the end of the https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule section.  It seems there are versions of cron (vixie) that take the day of the month and day of the week and 'or' them instead of 'and'.  Maybe just pick two days of the month:
```suggestion
      cronjob: "0 0 1,15 * *"  # Twice a month
```

---

By that logic, wouldn't `cronjob: "0 0 1,15 * *" # Twice a month`
Mean on day 1 or day 15 or any day of the month?